package.json

```diff
@@ -87,24 +87,24 @@
     "karma-spec-reporter": "0.0.26",
     "karma-webpack": "^1.7.0",
     "postcss-cssnext": "^2.4.0",
-    "postcss-import": "^8.0.2",
+    "postcss-import": "^8.1.0",
     "postcss-loader": "^0.9.1",
     "raf": "^3.2.0",
-    "ramda": "^0.21.0",
+    "ramda": "^0.22.1",
     "raw-loader": "^0.5.1",
     "reflect-metadata": "0.1.3",
-    "rimraf": "^2.5.1",
-    "rxjs": "5.0.0-beta.6",
+    "rimraf": "^2.5.4",
+    "rxjs": "5.0.0-beta.11",
     "selenium-standalone": "^5.1.0",
     "style-loader": "^0.13.0",
-    "stylelint": "^6.3.2",
-    "stylelint-webpack-plugin": "^0.2.0",
+    "stylelint": "^6.5.0",
+    "stylelint-webpack-plugin": "^0.3.1",
     "to-string-loader": "^1.1.3",
     "ts-helpers": "^1.1.1",
-    "ts-loader": "^0.8.2",
-    "tslint": "^3.3.0",
+    "awesome-typescript-loader": "^2.2.1",
+    "tslint": "^3.12.1",
     "tslint-loader": "^2.1.0",
-    "typescript": "^1.8.10",
+    "typescript": "^2.0.0",
     "typings": "^1.0.4",
     "webpack": "^1.13.0",
     "webpack-dev-server": "^1.14.1",
```

Comment:
I am still working on the webpack part since there is an error when updating to webpack 2.